content/en/docs/setup/upgrade/_index.md

```diff
@@ -7,5 +7,5 @@ test: n/a
 ---
 
 {{< warning >}}
-Upgrading across more than two minor versions (e.g., `1.6.x` to `1.9.x`) in one step is not officially tested or recommended.
+Upgrading across more than one minor version (e.g., `1.15.x` to `1.17.x`) in one step is not officially tested or recommended.
```

Comment:
```suggestion
Upgrading across more than two minor version (e.g., `1.15.x` to `1.17.x`) in one step is not officially tested or recommended. Upgrades that skip a minor version should only be done when using [Canary Upgrades](https://istio.io/latest/docs/setup/upgrade/canary/).
```

Per John's guidance

Reply:
+1 should be supported for canary

Reply:
We do test some upgrade testing across version although I need to verify what it's doing:  https://github.com/istio/istio/tree/master/tests/integration/helm/upgrade

Reply:
Should we say - Upgrading across more than two minor version (e.g., `1.15.x` to `1.17.x`) in one step is not officially tested or recommended for in place upgrade.

Reply:
Personally would like to specifically call out Canary Upgrades and link to doc to help guide toward a set procedure that is better practice.